images/installer/Dockerfile.rhel7

```diff
@@ -10,46 +10,50 @@ LABEL name="openshift3/ose-ansible" \
       summary="$SUMMARY" \
       description="$DESCRIPTION" \
       url="https://github.com/openshift/openshift-ansible" \
-      io.k8s.display-name="openshift-ansible" \
-      io.k8s.description="$DESCRIPTION" \
-      io.openshift.expose-services="" \
-      io.openshift.tags="openshift,install,upgrade,ansible" \
       com.redhat.component="aos3-installation-docker" \
-      version="v3.6.0" \
-      release="1" \
       architecture="x86_64"
 
 # Playbooks, roles and their dependencies are installed from packages.
-# Unlike in Dockerfile, we don't invoke the 'assemble' script here
-# because all content and dependencies (like 'oc') is already
-# installed via yum.
 USER root
 RUN INSTALL_PKGS="atomic-openshift-utils atomic-openshift-clients python-boto skopeo openssl java-1.8.0-openjdk-headless httpd-tools" && \
     yum repolist > /dev/null && \
     yum-config-manager --enable rhel-7-server-ose-3.6-rpms && \
     yum-config-manager --enable rhel-7-server-rh-common-rpms && \
+    yum install -y --setopt=tsflags=nodocs ansible && \
     yum install -y $INSTALL_PKGS && \
     yum clean all
 
-# The symlinks below are a (hopefully temporary) hack to work around the fact that this
-# image is based on python s2i which uses the python27 SCL instead of system python,
-# and so the system python modules we need would otherwise not be in the path.
-RUN ln -s /usr/lib/python2.7/site-packages/{boto,passlib} /opt/app-root/lib64/python2.7/
 
+RUN mkdir -p ${APP_HOME} ${APP_ROOT}/etc ${APP_ROOT}/bin
+RUN chmod -R ug+x ${APP_ROOT}/bin ${APP_ROOT}/etc /tmp/user_setup && \
+    /tmp/user_setup
+
+# Add entrypoint and other setup scripts
+COPY ./bin/user_setup /tmp
+COPY ./bin/ /usr/bin
+
+# Add files for running as a system container
+COPY ./system-container/root /
+
+# Back to the UID used in the base image
 USER ${USER_UID}
 
+ENV APP_ROOT=/opt/app-root
+ENV USER_NAME=default \
+    USER_UID=1001 \
+    APP_HOME=${APP_ROOT}/src \
+    HOME=${APP_ROOT}/src \
+    PATH=$PATH:${APP_ROOT}/bin
+
 # The playbook to be run is specified via the PLAYBOOK_FILE env var.
 # This sets a default of openshift_facts.yml as it's an informative playbook
 # that can help test that everything is set properly (inventory, sshkeys).
-# As the playbooks are installed via packages instead of being copied to
-# $APP_HOME by the 'assemble' script, we set the WORK_DIR env var to the
-# location of openshift-ansible.
 ENV PLAYBOOK_FILE=playbooks/byo/openshift_facts.yml \
     ANSIBLE_CONFIG=/usr/share/atomic-openshift-utils/ansible.cfg \
     WORK_DIR=/usr/share/ansible/openshift-ansible \
     OPTS="-v"
 
-# Add files for running as a system container
-COPY system-container/root /
+WORKDIR ${APP_HOME}
 
-CMD [ "/usr/libexec/s2i/run" ]
+RUN sed "s@${USER_NAME}:x:${USER_UID}:0@${USER_NAME}:x:\${USER_ID}:\${GROUP_ID}@g" /etc/passwd > ${APP_ROOT}/etc/passwd.template
```

Comment:
This `RUN` is now replaced by the entrypoint, which isn't defined... so this line should be replaced with `ENTRYPOINT ["/usr/bin/entrypoint"]`